### DISCUSSION TITLE
How can I set defaultValues from redux?

### DISCUSSION THREAD

**Discussion by nani1008** [react-hook-form/react-hook-form#5600] [Q&A]
So I have a table and if I click the table row it will prompt to another screen and fetch a data from api using ID after it, I want to display the values from redux which is the fetched data into my form, but as of now it is not displaying? Is this possible? or not? if yes what should I do? Thank you!!

**Comment by bluebill1049** [comment]
use reset api to update your form.

**Reply by nani1008** [reply]
reset api or setValue? I cant see on the docs that will display my defaultValues using reset api, tried` reset(defaultValues)` still not working

**Reply by bluebill1049** [reply]
provide a codesandbox.
